integration

```diff
@@ -594,6 +594,7 @@
   "mampfes/hacs_dwd_pollenflug",
   "mampfes/hacs_waste_collection_schedule",
   "mandarons/ha-bouncie",
+  "MapleEve/lifesmart-for-homeassistant",
```

Comment:
Missing country: <https://github.com/MapleEve/lifesmart-for-homeassistant/blob/main/hacs.json>